eoc_journal/utils.py

```diff
@@ -3,10 +3,19 @@
 from edx_rest_api_client.client import EdxRestApiClient
 
 try:
-    from openedx.core.lib.token_utils import JwtBuilder  # pylint: disable=F0401
+    # compatible with openedx ironwood and above
+    from openedx.core.djangoapps.oauth_dispatch.jwt import create_jwt_for_user
+
+    def create_jwt(user, scopes, expires_in):
+        return create_jwt_for_user(user)
+
 except ImportError:
-    JwtBuilder = None  # pylint: disable=C0103
+    # compatible with older openedx versions
+    # TODO: remove when ginkgo/hawthorn support is dropped
+    from openedx.core.lib.token_utils import JwtBuilder  # pylint: disable=F0401
 
+    def create_jwt(user, scopes, expires_in):
+        return JwtBuilder(user).build_token(scopes, expires_in)
```

Comment:
We generally try to keep compatibility patches in a single compat.py file so all compatibility stubs are in one place. 

In this case though, I think you can just assume Ironwood and remove backwards compatibility. Older versions can still use the older release, and the solutions fork only has one user so there won't ever be multiple releases to support.

Reply:
Done